extensions/wikia/ImageReview/ImageCountGetter.class.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+class ImageCountGetter extends WikiaModel {
```

Comment:
I'll second the notation the the caching was very often producing incorrect results. This looks good 👍